README.md

```diff
@@ -1 +1,140 @@
-# Budget-app
\ No newline at end of file
+# Budget App
+<a name="readme-top"></a>
+
+# 📗 Table of Contents
+
+- [📖 About the Project](#about-project)
+  - [🛠 Built With](#built-with)
+    - [Tech Stack](#tech-stack)
+    - [Live Demo](#live-demo)
+    - [Key Features](#key-features)
+- [💻 Getting Started](#getting-started)
+  - [Setup](#setup)
+  - [Prerequisites](#prerequisites)
+  - [Install](#install)
+  - [Usage](#usage)
+  - [Run tests](#run-tests)
+- [👥 Author](#author)
+- [🔭 Future Features](#future-features)
+- [🤝 Contributing](#contributing)
+- [⭐️ Show your support](#support)
+- [🙏 Acknowledgements](#acknowledgements)
+- [📝 License](#license)
+
+# 📖 [Budget app] <a name="about-project"></a>
+
+The **Budget App** is a Ruby on Rails mobile web application where you can manage your budget: you have a list of transactions associated with a category so that you can see how much money you spent and on what.
+
+## 🛠 Built With <a name="built-with"></a>
+
+Ruby, Ruby on Rails, Postgresql, Visual Studio Code.
+
+### Tech Stack <a name="tech-stack"></a>
+
+<details>
+  <summary>Ruby on Rails</summary>
+</details>
+
+<details>
+  <summary>PostgreSQL</summary>
+</details>
+
+## 🚀 Live Demo <a name="live-demo"></a>
+
+- ✅ Click [Live Demo Link](https://mic-0dbi.onrender.com)
```

Comment:
Great job on addressing the previous feedback! Just one small suggestion:

- [ ] Currently, there is a server-side error happening in your project. From my experience, it seems there might be an issue with the database migration on the deployed link. I recommend checking the remote database to ensure that the necessary tables exist. You can follow the [official documentation](https://docs.render.com/deploy-rails) for your chosen deployment method to troubleshoot and complete the migration process.

Keep up the good work! 🚀👍